### PR TITLE
Iss-Locator - 

### PR DESCRIPTION
**Pull request by victoriaaoka**
#### What does this PR do?
- Add express routes for the ISS-Locator app
- Create a user model for a Mongodb collection - users
- Create pug files for the app
- Fetch data from Open Notify and Google Maps
- Display a Google Map based on the data received from Open Notify
- Display ISS pass-times based on the data received from Open Notify
- Manage user authentication

#### Description of Task to be completed?
Show the point on a [Google Map](https://cloud.google.com/maps-platform/) directly below the current position of the [International Space Station](http://open-notify.org/Open-Notify-API/ISS-Location-Now/).

#### How should this be manually tested?
To test, Clone the repo and setup the project as outlined in the [Readme](https://github.com/victoriaaoka/iss-locator/blob/pre-dev/README.md) then run the command ``` $npm test```

#### Any background context you want to provide?
This is a capstone project with regard to the Andela JS Restacking course.

#### What are the relevant pivotal tracker stories?
The Pivotal Tracker board for this project can be found [here](https://www.pivotaltracker.com/n/projects/2204723)

#### Deployment
The app has been deployed on Heroku and can be found [here](http://iss-locator-staging.herokuapp.com/)

### CODE REVIEW

File: public/scripts/getissdata.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+//Get ISS data from Open Notify and pass it to the url.
+const button = document.getElementsByClassName('showbtn')[0];
+button.addEventListener('click', () => {
+  fetch(`http://api.open-notify.org/iss-now.json`).then(response => {
```
Comment (n8e) [inline public/scripts/getissdata.js]: The indentation here seems different from the rest of the app. Would you be willing to consider using `eslint`?

File: models/user.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+      /[0-9]/g.test(user.password) && 
+      /\w{6,}/.test(user.password) === true){
+      bcrypt.hash(user.password, 10, function(err, hash){
+        if(err){
+          return next(err);
+        }
+        user.password = hash;
+        next();
+    });
+    }else{
```
Comment (n8e) [inline models/user.js]: Should be a new line

**Review by n8e — CHANGES_REQUESTED** [review]
Please take a look at installing `eslint`. You can use a popular style guide when running `eslint --init`. It will help highlight the style issues in the code.

Otherwise looks good 👍

File: public/scripts/getissdata.js (hunk 1 above)
Comment (victoriaaoka) [reply public/scripts/getissdata.js]: Sure. I've just installed it and will be making changes asap.

**Pull request merged**